src/openforms/contrib/kadaster/config_check.py

```diff
@@ -39,3 +34,34 @@ def get_config_actions() -> list[Action]:
                 ),
             ),
         ]
+
+
+class BAGCheck:
+    verbose_name: str = _("Kadaster API: BAG")  # type: ignore
+
+    @staticmethod
+    def check_config():
+        try:
+            with get_bag_client() as client:
+                client.get_address("1000AA", "1", reraise_errors=True)
+        except NoServiceConfigured as exc:
+            msg = _("{api_name} endpoint is not configured.").format(
+                api_name="bag_service"
+            )
+            raise InvalidPluginConfiguration(msg) from exc
+        except requests.RequestException as exc:
+            raise InvalidPluginConfiguration(
+                _("Invalid response: {exception}").format(exception=exc)
+            ) from exc
+
+    @staticmethod
+    def get_config_actions() -> list[Action]:
+        return [
+            (
+                gettext("Configuration"),
+                reverse(
+                    "admin:bag_bagconfig_change",
```

Comment:
I get in the admin:
```
Interne fout: Reverse for 'bag_bagconfig_change' not found. 'bag_bagconfig_change' is not a valid view function or pattern name. 
```

Reply:
ah right, that one is gone indeed! weird that it didn't show for :thinking: